app/controllers/candidate_interface/start_page_controller.rb

```diff
@@ -26,5 +26,38 @@ def eligibility_params
       params.fetch(:candidate_interface_eligibility_form, {}).permit(:eligible_citizen, :eligible_qualifications)
         .transform_values(&:strip)
     end
+
+  private
+
+    def redirect_signed_in_candidate
+      if current_candidate.present?
+        add_course_from_find_id_to_candidate if params[:providerCode].present?
+
+        service = ExistingCandidateAuthentication.new(candidate: current_candidate)
```

Comment:
Would love any thoughts on what this service should be renamed to. The current naming doesn't really describe what is it doing